inventory/all_projects/orangelight

```diff
@@ -19,3 +12,14 @@ catalog3.princeton.edu
 catalog-indexer1.princeton.edu
 catalog-indexer2.princeton.edu
 catalog-indexer3.princeton.edu
+
+[orangelight_qa]
+catalog-qa1.princeton.edu
+
+[orangelight_staging]
+# catalog-staging1.princeton.edu
+catalog-staging2.princeton.edu
+catalog-indexer-staging1.princeton.edu
```

Comment:
I understand that we dint't have catalog-indexer-staging1 and this is the first staging indexer, but  can we name it to catalog-indexer-staging2 for consistency with the web server name?

Reply:
I will be making catalog-staging1 jammy shortly. So we will have two webservers and one indexer. I left catalog-staging1 in the event that things went kablooie